admin/fleetcommander/utils.py

```diff
@@ -37,11 +37,12 @@ def parse_config(config_file):
     SECTION_NAME = 'admin'
 
     args = {
-        'host': 'localhost',
```

Comment:
I'm afraid I committed this by mistake, we should prolly remove this and squash it